python_modules/dagster/dagster_tests/api_tests/test_api_get_current_runs.py

```diff
@@ -9,28 +11,33 @@
 from .utils import get_bar_repo_repository_location
 
 
+def create_dummy_test_run(instance: DagsterInstance, job_name: str) -> DagsterRun:
```

Comment:
there has to be N copies of approximately this function throughout the test suite - keep an eye out for a way to consolidate as you continue refactoring

Reply:
good call there was already a helper 👍🏻